Orange/statistics/util.py

```diff
@@ -355,22 +355,24 @@ def weighted_mean():
         return np.column_stack((
             np.nanmin(X, axis=0),
             np.nanmax(X, axis=0),
-            np.nanmean(X, axis=0) if not weighted else weighted_mean(),
-            np.nanvar(X, axis=0) if compute_variance else \
+            nanmean(X, axis=0) if not weighted else weighted_mean(),
+            nanvar(X, axis=0) if compute_variance else \
```

Comment:
`column_stack` is still not supported in dask. This means it will compute everything immediately and raise an ugly FutureWarning. If this is something that bothers us we could use something like `np.stack(..., axis=1)` instead.

Also if I am not mistaken `nanmean` calls bottleneck's function for non sparse arrays, which causes trouble with dask arrays, right?

While I'm lookimg at this function `weighted_mean` also calls bottleneck's `nansum` (same concern as above) as well as `np.c_[weights]` (I do not see the point of this).